website/docs/events/m1.md

```diff
@@ -40,7 +42,273 @@ Please consider topics such as:
 
 During the workshop, participants will put their ideas into practice and take the first steps towards having a full-fledged class that uses modern technology for efficient teaching.
 
-## Open Discussion over Lunch: 13:00 - 14:00
+### Open Discussion over Lunch: 13:00 - 14:00
 
 Lunch is on the Open Education Hub team.
 As always, lunch will be a pretext for more open-ended discussions regarding the Open Education Hub project and the guests' further involvement with it. 
+
+## Summary
+
+Find the entire discussion from our meeting below in text format or on [YouTube](https://www.youtube.com/watch?v=RkUnCXL_BtA&ab_channel=OpenEducationHub) in audio format (Romanian only).
+
+Speakers:
+* **RD:** Răzvan Deaconescu
+* **RN:** Răzvan Nițu
+* **TD:** Teodor Duțu
+* **LB:** Liza Babu
+* **SW:** Sergiu Weisz
+
+Audience:
+* **AO:** Andrei Olaru
+* **DN:** Dan Novischi
+* **BB:** Bogdan Bocșe
+* **GC:** Gabriela Ciuprina
+
+### Project Overview
+
+**RD:** We want to see how realistic is our idea.
+
+**RD:** Our content is often on the Wiki, but is not easily reviewed / updated.
+
+**RD:** I had problems.
+When suggestions arise, people send the fix to you and you have to deal with it.
+We want to fix this.
+
+**RD:** We don't have a collaborative way to do it.
+
+**RD:** Folks from Singapore are using our content as it is, they don't have a what to update it for their needs.
+
+**RD:** A good example of how it's supposed to be is the Operating Systems 2 repository.
+It's a repository based on the Linux Kernel repository that has our deployment infrastructure on top.
+Everything works in a VM, it is easy to use, update, and create snapshots.
+Students can use it as it is, and can come up with fixes via pull requests.
+
+**RD:** This was our motivation.
+
+**RD:** We usualy receive feedback from students that we have outdated content, information that is repetitive or even mistakes.
+They do not have write access to OCW and they cannot be granted write access because OCW does not have a method for reviewing materials.
```

Comment:
```suggestion
They do not have write access to [OCW (our wiki)](https://ocw.cs.pub.ro/courses/) and they cannot be granted write access because OCW does not have a method for reviewing materials.
```